Documentation/contributing/testing/e2e.rst

```diff
@@ -507,12 +507,17 @@ cluster.
 
 ::
 
-  CNI_INTEGRATION=gke K8S_VERSION=1.17 ginkgo --focus="K8sDemo" -- -cilium.provision=false -cilium.kubeconfig=`echo ~/.kube/config` -cilium.image="quay.io/cilium/cilium-ci" -cilium.operator-image="quay.io/cilium/operator" -cilium.operator-suffix="-ci" -cilium.hubble-relay-image="quay.io/cilium/hubble-relay-ci" -cilium.passCLIEnvironment=true
+  CNI_INTEGRATION=gke K8S_VERSION=1.17 NATIVE_CIDR=$NATIVE_CIDR ginkgo --focus="K8sDemo" -- -cilium.provision=false -cilium.kubeconfig=`echo ~/.kube/config` -cilium.image="quay.io/cilium/cilium-ci" -cilium.operator-image="quay.io/cilium/operator" -cilium.operator-suffix="-ci" -cilium.hubble-relay-image="quay.io/cilium/hubble-relay-ci" -cilium.passCLIEnvironment=true
 
 .. note:: The kubernetes version defaults to 1.18 but can be configured with
           versions between 1.13 and 1.18. Version should match the server
           version reported by ``kubectl version``.
 
+.. note:: The correct ``NATIVE_CIDR`` value can be obtained, as described in the
+          :ref:`k8s_install_gke` document, with the
+          ``gcloud container clusters describe $CLUSTER_NAME --zone $CLUSTER_ZONE --format 'value(clusterIpv4Cidr)')``
```

Comment:
Would be great to have this line _above_ the ginkgo command. Otherwise, people may copy-paste the ginkgo command and fail to pass the native CIDR correctly, come back to the doc, search for the step they missed, etc.

What about inserting a step, or even simpler, just adding this command to the code block for the ginkgo command?

Reply:
Added the command to the code block :+1: 
I also kept the note (moved above) to have some more context on why those commands are needed